Source/Core/Core/IPC_HLE/WII_IPC_HLE_Device_usb_ven.cpp

```diff
@@ -15,21 +15,19 @@ CWII_IPC_HLE_Device_usb_ven::CWII_IPC_HLE_Device_usb_ven(const u32 device_id,
 IPCCommandResult CWII_IPC_HLE_Device_usb_ven::IOCtlV(const IOSIOCtlVRequest& request)
 {
   request.Dump(GetDeviceName());
-  request.SetReturnValue(IPC_SUCCESS);
   return GetNoReply();
 }
 
 IPCCommandResult CWII_IPC_HLE_Device_usb_ven::IOCtl(const IOSIOCtlRequest& request)
 {
   request.Log(GetDeviceName(), LogTypes::OSHLE);
-  request.SetReturnValue(IPC_SUCCESS);
 
-  IPCCommandResult reply = GetDefaultReply();
+  IPCCommandResult reply = GetDefaultReply(IPC_SUCCESS);
```

Comment:
Since most of those are `IPC_SUCCESS` anyways, we could remove the duplicate initializations to the same value...but...it is more verbose that way (ie. you see what you get) - so, I'm fine either way, just looking what others think.
And it looks like`USBV5_IOCTL_GETDEVICECHANGE` could use some more cleanup (not necessarily here tho), as `firstcall` looks fairly useless now and that `return reply` there is inconsistent with the rest of the cases.